build.gradle

```diff
@@ -127,8 +128,10 @@ configure(allprojects) { project ->
 
   dependencies {
     compileOnly "com.google.code.findbugs:jsr305:$googleJsr305Version"
+    compileOnly "io.micrometer:micrometer-core:$micrometerVersion"
```

Comment:
Not `optional`?

Reply:
This version of gradle doesn't seem to support it; we really need to upgrade to a recent version but my gradle/groovy skills are not up to it.

Reply:
Yes but a lot has changed since then `compile` etc have been deprecated - there are other changes too - e.g. maven publishing.

Reply:
@garyrussell @artembilan upgrading this project to Gradle 7.4 is on my todo list. made a bit difficult by the fact that it lagged on 5.x indeed...